cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -7202,24 +7202,34 @@ impl LowerBackend for X64Backend {
 
                 Opcode::BrTable => {
                     let jt_size = targets.len() - 1;
-                    assert!(jt_size <= u32::max_value() as usize);
+                    assert!(jt_size <= u32::MAX as usize);
                     let jt_size = jt_size as u32;
 
+                    let ty = ctx.input_ty(branches[0], 0);
+                    let ext_spec = match ty {
+                        types::I128 => panic!("BrTable unimplemented for I128"),
```

Comment:
You will still have to keep the special case in `cranelift/frontend/src/switch.rs` for 128bit ints.